dtrace_probe.cc

```diff
@@ -59,8 +61,17 @@ namespace node {
       return Nan::Undefined();
     }
 
+    int cblen = argsinfo.Length() - fnidx - 1;
```

Comment:
Should `cblen` be checked to make sure it's >= 0 by throwing an error and returning `Nan::Undefined()` otherwise?